docs/contributing-to-airbyte/README.md

```diff
@@ -44,6 +45,13 @@ We're not actively reviewing/accepting new Java connectors for now.
 10. Review and respond to feedback and questions by Airbyte maintainers.
 11. Merge the contribution.
 
+You can check the status of your contribution in this [Github Project](https://github.com/orgs/airbytehq/projects/108/views/4). It will provide you what Sprint your contribution was assigned and when you can expect a review.
+
+:::warning
```

Comment:
Unrelated: I wonder if we can make a workflow that posts a comment and closes such PRs. Can you link a guide / note that explains why we have this limitation?